cloud-common/src/main/java/com/sequenceiq/cloudbreak/polling/ExtendedPollingResult.java

```diff
@@ -56,6 +65,11 @@ public ExtendedPollingResultBuilder withPayload(Set<Long> failedInstanceIds) {
             return this;
         }
 
+        public ExtendedPollingResultBuilder withPayloadWithHostNames(Set<String> failedHostNames) {
+            this.failedHostNames = failedHostNames;
+            return this;
+        }
```

Comment:
field name and method name are out of sync. though i can already see it for failedInstanceIds, cann't we name it as withFailedHostNames